Node/package.json

```diff
@@ -23,7 +23,6 @@
   "main": "./lib/botbuilder-azure.js",
   "typings": "./lib/botbuilder-azure.d.ts",
   "dependencies": {
-    "@types/tedious": "^1.8.31",
```

Comment:
Why is it unneeded? When I try to install this module it complains with following:

```
[Typescript] node_modules/@sheerun/botbuilder-azure/lib/botbuilder-azure.d.ts(1,53): error TS7016: Could not find a declaration file for module 'tedious'. '/messages/node_modules/tedious/lib/tedious.js' implicitly has an 'any' type.
[Typescript]   Try `npm install @types/tedious` if it exists or add a new declaration (.d.ts) file containing `declare module 'tedious';
```

Only adding `@types/tedious` to main project solves this which is not ideal.

Also see:

https://stackoverflow.com/questions/45176661/how-do-i-decide-whether-types-goes-into-dependencies-or-devdependencies

It says: As soon as the type appears in declaration file, you need to place it on dependencies. Otherwise devDependencies is fine

It happens you've put these types in declaration file, so you need to use dependencies instead of devDepenedncies.